Draft of learner profiles (#54)
czi-catalystproject/hub-champion-training: profiles/learner-profiles.md

# Learner Profiles

> ## María Zambrano
> She graduated in Computer Science recently and has a support staff position in the biomedical engineering PhD program at the life sciences department in a medium-sized public university in Ecuador.

Reviewer: ```suggestion
## María Zambrano

She graduated in Computer Science recently and has a support staff position in the biomedical engineering PhD program at the life sciences department in a medium-sized public university in Ecuador.
```

> María has to assist around 25 students and their advisors in struggling with hardware that is not necessarily old but does not hold the capacity to process the required “big data” for their projects in a timely manner. Also, the projects are very different, so there isn’t a clear one-size-fits-all solution.

Reviewer: ```suggestion
María has to assist around 25 students and their advisors in struggling with hardware that is not necessarily old but does not hold the capacity to process the required “big data” for their projects promptly. Also, the projects are very different, so there isn’t a clear one-size-fits-all solution.
```

> The Hub Champion training will able María to select and recommend the best environments for the students' project needs.

Reviewer: ```suggestion
The Hub Champion training will enable María to select and recommend the best software environments and computation resources for the students' project needs.
```

## Santiago Hernández

> He is a biologist with some self-taught knowledge of computer science. He is part of a nonprofit organization working mainly in Mexico but with an internationally distributed team. The team has folks with biology, pharmacy, media studies, and political science. 

Reviewer: ```suggestion
He is a biologist with some self-taught knowledge of Computer Science. He is part of a nonprofit organization working mainly in Mexico but with an internationally distributed team. The team has cross-disciplinary intersections with biology, pharmacy, media studies, and political science. 
```

> ## Santiago Hernández
> He is a biologist with some self-taught knowledge of Computer Science. He is part of a nonprofit organization working mainly in Mexico but with an internationally distributed team. The team has cross-disciplinary intersections with biology, pharmacy, media studies, and political science. 

Reviewer: ```suggestion
## Santiago Hernández

He is a biologist with some self-taught knowledge of Computer Science. He is part of a nonprofit organization working mainly in Mexico but with an internationally distributed team. The team has cross-disciplinary intersections with biology, pharmacy, media studies, and political science. 
```

The community members have different levels of knowledge of interactive computing tools. Because of his better understanding of programming and computing concepts, Santiago was selected as the Hub Champion for the Catalyst Project.

> The Hub Champion training will give Santiago more confidence and tools to provide a better use of the hub to his community while he connects with other nonprofit organizations with similar challenges to share strategies.

Reviewer: ```suggestion
The Hub Champion training will give Santiago more confidence and tools to make the best use of the hub in his community while he connects with other nonprofit organizations with similar challenges to share strategies.
```

## Ana Muñoz

> She is a Biochemistry working in analyzing water quality on the coast of Peru. She is part of a climate data research group distributed on the Pacific Latin American coast, co-led by two other professors from public and private universities in Chile and Colombia.

Reviewer: ```suggestion
She is a biochemist working on analyzing water quality on the coast of Peru. She is part of a climate data research group distributed on the Pacific Latin American coast, co-led by two other professors from public and private universities in Chile and Colombia.
```

> ## Ana Muñoz
> She is a biochemist working on analyzing water quality on the coast of Peru. She is part of a climate data research group distributed on the Pacific Latin American coast, co-led by two other professors from public and private universities in Chile and Colombia.

Reviewer: ```suggestion
## Ana Muñoz

She is a biochemist working on analyzing water quality on the coast of Peru. She is part of a climate data research group distributed on the Pacific Latin American coast, co-led by two other professors from public and private universities in Chile and Colombia.
```

The project aims to understand the effects of El Niño on human health. They have been using each university’s computational resources for research, so they often use distinct tools and protocols and lack shared infrastructure and similar training. As part of the Catalyst Project, they want to develop a common workflow across the three universities and find a way to sustain the project for the next two decades due to the long timelines used in climate data.

> The Hub Champion training will give Ana the knowledge to establish a workflow that can run on open-source software giving sustainability to the research project.

Reviewer: ```suggestion
The Hub Champion training will give Ana the knowledge to establish a workflow using open-source software that enables collaboration and facilitates reproducibility for the research project.
```
